### ISSUE TITLE
is this method capable of performing zero shot?

### ISSUE THREAD

**Issue by francqz31** [fenglinglwb/PSM#5]
Hello , I wanted to ask about if this method is capable of performing zero shot image face mask inpainting on pics that are not included in the training data at the same high quality like the ones presented in the paper for example like this pic from (https://github.com/andreas128/RePaint) repository .
![150849757-5cd762cb-07a3-46aa-a906-0s](https://user-images.githubusercontent.com/117993423/228623149-b1ddddc1-41e5-487a-98e9-37c8c6ec6cf2.jpg)
